test.rb

```diff
@@ -53,23 +54,24 @@
 
 puts 'my_count'
 ary = [1, 2, 4, 2]
-puts ary.count               #=> 4
-puts ary.count(2)            #=> 2
-puts ary.count{ |x| x%2==0 } #=> 3
+puts ary.my_count               #=> 4
+puts ary.my_count(2)            #=> 2
+puts ary.my_count{ |x| x%2==0 } #=> 3
```

Comment:
**_#my_count counts the number of items in enum that are equal to an item if an argument is given_**